src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$daySize: 100px;
+$countDays: 31;
+$gapValue: 1px;
+$calendarWidth: $daySize * 7 + 10;
```

Comment:
The calculation for `$calendarWidth` seems incorrect. The `10` added to `$daySize * 7` might not be the intended gap or margin. Consider using `$gapValue` if the intention is to add a gap between days.